<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductPage</class>
 <widget class="QWidget" name="ProductPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>商品详情</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget#ProductPage {
    background: qlineargradient(spread:pad,x1:0,y1:0,x2:1,y2:1,
               stop:0 #ede9fb, stop:0.45 #e0d4f8, stop:1 #c5b3ef);
    border-radius:20px;
    font-family:&quot;微软雅黑&quot;;
    color:#4f407a;
}
#widget_titleBar {
    background: qlineargradient(spread:pad,x1:0,y1:0,x2:1,y2:0,
               stop:0 #d8c8f3, stop:1 #c4b0ec);
    border-top-left-radius:20px;
    border-top-right-radius:20px;
}
QLabel#lab_title {
    font: 600 19px &quot;微软雅黑&quot;;
    color:#4a396d;
    letter-spacing:1px;
}
QToolButton#btn_close, QToolButton#btn_fav {
    background: rgba(255,255,255,0.55);
    border:1px solid rgba(140,110,190,0.35);
    border-radius:14px;
    width:28px;
    height:28px;
    font: 700 14px &quot;微软雅黑&quot;;
    color:#5a4588;
}
QToolButton#btn_close:hover { background:#ff6b81; color:#fff; border-color:#ff6b81; }
QToolButton#btn_fav[checked=&quot;true&quot;] {
    background:#ffce5d;
    color:#744f00;
    border-color:#e0a93d;
}

QLabel#lab_pic {
    background:#f7f5fc;
    border:2px solid #e2daf6;
    border-radius:18px;
}

QLabel#lab_name {
    font: 700 26px &quot;微软雅黑&quot;;
    color:#5c477f;
}
QLabel#lab_price {
    font: 700 24px &quot;微软雅黑&quot;;
    color:#a46be2;
}
QLabel#lab_amount, QLabel#lab_sales {
    font: 12pt &quot;微软雅黑&quot;;
    color:#6b588a;
}
QLabel#lab_about {
    background: rgba(255,255,255,0.85);
    border:1px solid #decff4;
    border-radius:14px;
    padding:12px 14px;
    font: 11.5pt &quot;微软雅黑&quot;;
    color:#4d3c70;
}

#widget_actions {
    background: transparent;
    border-bottom-left-radius:20px;
    border-bottom-right-radius:20px;
}

QPushButton.actionBtn {
    background: qlineargradient(spread:pad,x1:1,y1:0,x2:0,y2:1,
               stop:0 #bfa8f2,stop:1 #a485dd);
    color:#fff;
    border:none;
    border-radius:18px;
    padding:10px 30px;
    font: 600 14px &quot;微软雅黑&quot;;
    min-width:130px;
}
QPushButton.actionBtn:hover {
    background:#9d76d6;
}
QPushButton.actionBtn:pressed {
    background:#885ec8;
}
QPushButton#btn_buy {
    background: qlineargradient(spread:pad,x1:0,y1:0,x2:1,y2:1,
               stop:0 #ffbb6b, stop:1 #ff9966);
}
QPushButton#btn_buy:hover { background:#ff9a43; }
QPushButton#btn_buy:pressed { background:#f27d1f; }

QPushButton#btn_closePanel {
    background:rgba(100,80,140,0.18);
    color:#4a396d;
    min-width:110px;
}
QPushButton#btn_closePanel:hover { background:rgba(100,80,140,0.32); }
QPushButton#btn_closePanel:pressed { background:rgba(100,80,140,0.50); }
   </string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="titleLayout">
     <property name="leftMargin">
      <number>16</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>12</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="lab_title">
       <property name="text">
        <string>商品详情</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="titleSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btn_fav">
       <property name="toolTip">
        <string>收藏 / 取消收藏</string>
       </property>
       <property name="text">
        <string>★</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_close">
       <property name="toolTip">
        <string>关闭</string>
       </property>
       <property name="text">
        <string>✕</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="contentLayout">
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>14</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QWidget" name="topInfo" native="true">
       <layout class="QHBoxLayout" name="topInfoLayout">
        <property name="spacing">
         <number>16</number>
        </property>
        <item>
         <widget class="QLabel" name="lab_pic">
          <property name="minimumSize">
           <size>
            <width>240</width>
            <height>260</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>240</width>
            <height>300</height>
           </size>
          </property>
          <property name="text">
           <string>图片</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_info" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_info">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="lab_name">
             <property name="text">
              <string>商品名称</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lab_price">
             <property name="text">
              <string>￥0.00</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lab_amount">
             <property name="text">
              <string>库存：0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lab_sales">
             <property name="text">
              <string>销量：0</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_about">
       <property name="text">
        <string>商品简介</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="actionsLayout">
     <property name="leftMargin">
      <number>26</number>
     </property>
     <property name="topMargin">
      <number>6</number>
     </property>
     <property name="rightMargin">
      <number>26</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_closePanel">
       <property name="styleSheet">
        <string notr="true">qproperty-class: actionBtn;</string>
       </property>
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="actionSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cart">
       <property name="styleSheet">
        <string notr="true">qproperty-class: actionBtn;</string>
       </property>
       <property name="text">
        <string>加入购物车</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_buy">
       <property name="styleSheet">
        <string notr="true">qproperty-class: actionBtn;</string>
       </property>
       <property name="text">
        <string>立即购买</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
